src/coreclr/vm/CMakeLists.txt

```diff
@@ -860,7 +868,22 @@ elseif(CLR_CMAKE_TARGET_ARCH_ARM64)
             ${ARCH_SOURCES_DIR}/singlestepper.cpp
         )
     endif(CLR_CMAKE_HOST_UNIX)
-elseif(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
+elseif(clr_cmake_target_arch_loongarch64)
+    set(vm_sources_dac_and_wks_arch
+        ${arch_sources_dir}/stubs.cpp
+        exceptionhandling.cpp
+    )
+
+    set(vm_headers_dac_and_wks_arch
+        ${arch_sources_dir}/virtualcallstubcpu.hpp
+        exceptionhandling.h
+    )
+
+    set(vm_sources_wks_arch
+        ${arch_sources_dir}/profiler.cpp
+        gcinfodecoder.cpp
+    )
```

Comment:
@clamp03  why did you destroy the LoongArch64's config?????

Reply:
@shushanhf  I am really sorry. It is just my mistake. I don't want to destroy LoongArch64 at all. Sorry again.

Reply:
> @shushanhf I am really sorry. It is just my mistake. I don't want to destroy LoongArch64 at all. Sorry again.

shake hands.